enhancements/sig-architecture/32-extensiblescheduling/32-extensiblescheduling.md

```diff
@@ -0,0 +1,321 @@
+  - On each managed cluster, there would be a customized agent to sum up the cpu usage of the cluster. Then update the value into the ManagedClusterScalar CR with name `customized-resourceratiocpu`.
+  - On the hub, the placement prioritizer will read value from ManagedClusterScalar `customized-resourceratiocpu` and score based on it. 
+
+#### Story 2: The placement controller could prioritize clusters by metrics collected on hub.
+  - As an end user, can define prioritizer name `customized-resourceratiocpu` in placment spec.prioritizerPolicy.configurations to say he wants to schedule based on customized value resourceratiocpu.
+  - On the hub, there would be a customized agent to read cpu usage from thanos. It will generate a value for each cluster and update it into ManagedClusterScalar CR with name `customized-resourceratiocpu`.
+  - On the hub, the placement prioritizer will read value from ManagedClusterScalar `customized-resourceratiocpu` and score based on it.
+
+#### Story 3: The placement controller could prioritize clusters for disaster recovery.
+  - As an end user, can define prioritizer name `customized-dr` in placment spec.prioritizerPolicy.configurations to say he wants to schedule workload based on disaster recovery score.
+  - On the hub, there would be a customized agent to give cluster in the frist DC the highest score.
+  - On the hub, the placement prioritizer will select the cluster in the frist DC. Then disaster recovery workload will be running on it.
+  - When this cluster is unavailable (a taint is added), the customized agent will give cluster in the second DC the highest value.
+  - On the hub, the placement prioritizer will re-schedule to select the cluster in the second DC. Then disaster recovery workload will be running on the second DC.
+
+### Goals
+- Design an new API(CRD) to contain a scalar value for each managed cluster.
+- Let placement prioritizer plugin support schedule with customized value.
+- Let placement controller maintain creating and cleaning new CR based on placement configurations.
+
+### Non-Goals
+- How the customized agent generate and update value into new CR.
+- Placement filter will not support customized value.
+
+## Proposal
+
+### 1. ManagedClusterScalar API
+ManagedClusterScalar is the new API we want to add, to represents a scalable value (aka score) of one managed cluster.
+```go
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:resource:scope="Namespaced"
+// +kubebuilder:subresource:status
+
+// ManagedClusterScalar represents a scalable value (aka score) of one managed cluster.
+// Each ManagedClusterScalar only represents the score for one specific calculator type.
+// ManagedClusterScalar is a namesapce scoped resource.
+//
+// The ManagedClusterScalar name should follow the format {cluster name}-{calculator name}.
```

Comment:
I do not think it needs {cluster name}, but it needs to mention here that the namespace of the resource is the cluster ns.